src/qibolab/designs/basic.py

```diff
@@ -1,8 +1,49 @@
 from dataclasses import dataclass, field
+from typing import List, Optional
 
 from qibolab.instruments.abstract import AbstractInstrument
 
 
+@dataclass
+class Channel:
+    """Representation of physical wire connection (channel)."""
+
+    name: str
+    """Name of the channel from the lab schematics."""
+
+    qubit: Optional["Qubit"] = field(default=None, repr=False)
+    """Qubit connected to this channel. Used to set read the sweetspot for flux channels only."""
+    ports: List[tuple] = field(default_factory=list)
+    """List of tuples (controller, port) connected to this channel."""
+    local_oscillator: Optional[AbstractInstrument] = None
```

Comment:
It can also be an empty class, it is just to have a different type:
```py
class LocalOscillator(AbstractInstrument):
    pass
```

Reply:
I implemented the empty `LocalOscillator` class. In fact, this class does not need to be empty and can implement most methods associated to local oscillators. For example, see #287 for a new LO driver, which is basically duplicating the existing Rohde-Schwarz code. So these methods could be moved to the generic `LocalOscillator` to avoid duplication.

However, I would leave this cleanup for later, as the Rohde-Schwarz driver is also used by other platforms that I am not testing with and I don't want to risk breaking them.